FutureNorthants/northants-website
# Add TLS check

### Testing

- Checkout this branch, `yarn install && yarn dev`
- View any page, should be no changes
- Now edit the `src/lib/tlsChecker.ts` file to change the TLS_CHECK_URL to anything else and refresh the page, expect a warning banner
- Back out your changes. Edit your env.local to set TLS_CHECK_URL to an empty string. Restart Next server and refresh the page, expect no warning banner.
